### ISSUE TITLE
Autodrive and Tardis teleport [BUG]

### ISSUE THREAD

**Issue by garo73** [sperrgebiet/FS19_Tardis#10]
AutoDrive Version: 1.1.1.2   
Release or Custom Zip: Release  
Map (If applicable): All  
Vehicle / Type (If applicable): All 
Multiplayer or Singleplayer: Singleplayer, maybe both

When I use Tardis Teleport (Latest version 0.9.1.7), the cam freeze and can not move... I have made the same issue at both author!

[log.txt](https://github.com/sperrgebiet/FS19_Tardis/files/6720271/log.txt)

**Comment by sperrgebiet** [comment]
Just pushed a new version which should fix the camera freeze. Seems I screwed some things up before and it would have never worked as intended. Nevertheless, I'd blame AD, just to feel better ;)

Seriously, I'm pretty confident that there is no regression. Let me know if you encounter any issues, so far it looks good on my side with the latest AD version.

https://github.com/sperrgebiet/FS19_Tardis/blob/master/FS19_Tardis.zip?raw=true

**Comment by garo73** [comment]
Ok! With the new Teleport, all works fine! Thanks to all, good Job!

**Comment by garo73** [comment]
No, I was wrong, from time to time and not always, it still does it but I can't understand in what conditions to create a repeatability ... I keep looking.

**Comment by garo73** [comment]
It seems to do so when I move something full, such as a trailer full of fruit attached to a truck/tractor, or a full combine harvester, or a tractor with a full fertilizer sprayer.
